packages/cf-component-pagination/src/PaginationItem.js

```diff
@@ -68,4 +92,8 @@ PaginationItem.propTypes = {
   children: PropTypes.node
 };
 
-export default PaginationItem;
+const PaginationItemUnstyled = createComponent(styles, PaginationItem);
+
+export { PaginationItemUnstyled };
+
+export default applyTheme(PaginationItemUnstyled, PaginationItemTheme);
```

Comment:
I want consistency with my code, I use default export for PaginationLink and PaginationRoot